packages/store/tests/state.spec.ts

```diff
@@ -163,4 +170,109 @@ describe('State', () => {
       expect(TestBed.get(Store).snapshot().foo).toEqual(['updateState', 'onInit']);
     });
   });
+
+  describe('"ngxsAfterBootstrap" lifecycle hook', () => {
+    function createRootNode(selector = 'app-root'): void {
+      const document = TestBed.get(DOCUMENT);
+      const adapter: DomAdapter = new BrowserDomAdapter();
+
+      const root = adapter.firstChild(
+        adapter.content(adapter.createTemplate(`<${selector}></${selector}>`))
+      );
+
+      const oldRoots = adapter.querySelectorAll(document, selector);
+      oldRoots.forEach(oldRoot => adapter.remove(oldRoot));
+
+      adapter.appendChild(document.body, root);
+    }
+
+    const enum LifecycleHooks {
+      NgOnInit = 'ngOnInit',
+      NgxsOnInit = 'ngxsOnInit',
+      NgAfterViewInit = 'ngAfterViewInit',
+      NgxsAfterBootstrap = 'ngxsAfterBootstrap'
+    }
+
+    let hooks: LifecycleHooks[] = [];
+
+    @Component({
+      selector: 'app-root',
+      template: ''
+    })
+    class MockComponent implements OnInit, AfterViewInit {
+      public ngOnInit(): void {
+        hooks.push(LifecycleHooks.NgOnInit);
+      }
+
+      public ngAfterViewInit(): void {
+        hooks.push(LifecycleHooks.NgAfterViewInit);
+      }
+    }
+
+    @NgModule({
+      imports: [BrowserModule],
+      declarations: [MockComponent],
+      entryComponents: [MockComponent]
+    })
+    class MockModule {
+      public static ngDoBootstrap(app: ApplicationRef): void {
+        createRootNode();
+        app.bootstrap(MockComponent);
+      }
+    }
+
+    beforeEach(() => (hooks = []));
+
+    it('should invoke "ngxsAfterBootstrap" after "ngxsOnInit" and after root component\'s "ngAfterViewInit"', () => {
+      @State({ name: 'foo' })
+      class FooState implements NgxsOnInit, NgxsAfterBootstrap {
+        public ngxsOnInit(): void {
+          hooks.push(LifecycleHooks.NgxsOnInit);
+        }
+
+        public ngxsAfterBootstrap(): void {
+          hooks.push(LifecycleHooks.NgxsAfterBootstrap);
+        }
+      }
+
+      TestBed.configureTestingModule({
+        imports: [MockModule, NgxsModule.forRoot([FooState])]
+      });
+
+      MockModule.ngDoBootstrap(TestBed.get(ApplicationRef));
+
+      expect(hooks).toEqual([
+        LifecycleHooks.NgxsOnInit,
+        LifecycleHooks.NgOnInit,
+        LifecycleHooks.NgAfterViewInit,
+        LifecycleHooks.NgxsAfterBootstrap
+      ]);
+    });
+
+    it('should invoke "ngxsAfterBootstrap" for lazy states', () => {
```

Comment:
```suggestion
    it('should invoke "ngxsAfterBootstrap" for feature states', () => {
```
This is more a test for feature states. Testing lazy states would require a whole dance with the router.

Reply:
Ocurrences of lazy in the test would have to change too ;-)